src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -59,19 +110,30 @@ def main(argv: Optional[Sequence[str]] = None) -> None:
     cfg_mgr = ConfigManager(config_file=args.config,
                             config_dir=args.config_dir,
                             check=args.check)
-
-    logging.info("ConfigManager: config_data = %s", repr(cfg_mgr.config_data))
+    try:
+        logger.info("ConfigManager: config_data = %s",
+                    repr(cfg_mgr.config_data))
+    except (ConfigManagerError, ConflictingBackendsError,
```

Comment:
Note that all of these exceptions are derived classes from a common parent exception class, namely `ConfigManagerException`, so instead of listing all of these specific exceptions as possible ones to catch, you can just catch the common ancestor exception, since they are all derived from it.

This will also make the code more maintainable going forward if we add additional ConfigManager releated exceptions; so long as they have `ConfigManagerException` as an ancestor, we wouldn't need to change this except statement to handle them.